src/main/java/nwoolcan/model/brewery/production/batch/step/ParameterTypeEnum.java

```diff
@@ -0,0 +1,37 @@
+package nwoolcan.model.brewery.production.batch.step;
+
+import nwoolcan.model.utils.UnitOfMeasure;
+
+/**
+ * Parameter type enum implementation.
+ */
+public enum ParameterTypeEnum implements ParameterType {
+    /**
+     * Temperature parameter type.
+     */
+    Temperature(UnitOfMeasure.Celsius),
+    /**
+     * Number of hops that can be added in some production steps.
+     */
+    AddedHops(UnitOfMeasure.Unit),
```

Comment:
Shouldn't we use mass instead of unit?

Reply:
A mass unit like Kilogram you mean? Can do